src/main/resources/swagger.api/schemas/tenant.yaml

```diff
@@ -47,3 +47,26 @@ TenantCollection:
   required:
     - tenants
     - totalRecords
+
+TenantDeleteRequest:
+    type: object
+    properties:
+      deleteType:
+        type: string
+        enum: [ "HARD", "SOFT" ]
+      deleteOptions:
+        type: object
+        properties:
+          deleteInternalData:
```

Comment:
lets remove deleteInternalData param from contract, because hard delete means that record from tenant table and all tables with tenantId foreigKey would be deleted, so this param is not needed